src/applications/check-in/components/pages/Appointments/index.jsx

```diff
@@ -65,6 +68,15 @@ const AppointmentsPage = props => {
     [setPrivacyActModalOpen],
   );
 
+  useEffect(
```

Comment:
This works pretty well for me and is probably `good enough` but it does cause the page to always be reloaded after check-in is complete. Not sure if that is even a problem, but if it is there are a few alternatives that we could do. But that said, I'm good with this if you are good with it!